Haskell/Haskell.sublime-syntax

```diff
@@ -187,20 +183,25 @@ contexts:
       scope: punctuation.separator.comma.haskell
   block_comment:
     - match: '\{-(?!#)'
-      captures:
-        0: punctuation.definition.comment.haskell
+      scope: punctuation.definition.comment.begin.haskell
       push:
         - meta_scope: comment.block.haskell
+        - match: '\{-#'
```

Comment:
ugh, not explaining myself very well, sorry. It is so that preprocesor directives inside a comment block are correctly treated as nested comments. for more info, see the linked issue :)